lib/form/paragraph3.flow

```diff
@@ -45,6 +45,20 @@ possible, and because rendering words in the same sprite improves the interword
 */
 
 
+// How extra space of paragraph line will be spread:
+// start — added at start of first word;
+// justify — equally distributed between words or added at the end(!) of the only word or last line;
+// end=1-start-justify — what's left will be at end of the line.
+// We might store start and end, but even this variant seems symmetrical,
+// the used one is more useful for calculations.
+GeneralAlignment(
+	start : double,
```

Comment:
using double to model 3  way value (0 / 0.5 / 1) looks as old C-style / Form-style, maybe there is better solution?

Reply:
Actually, I can't see any benefits from introducing GeneralAlignment. The code in general becomes more obfuscated from my point of view, more odd checks are required now.
Do we really need it? what are the benefits?

Reply:
The odd check is about justify, I will correct that. Other is pure arithmetics, we have exactly the same for TAlign.

Reply:
aligning the code with tropic is good indeed